### DISCUSSION TITLE
feedback: development experience on linux

### DISCUSSION THREAD

**Discussion by RomanMIzulin** [enso-org/enso#11002] [Ideas and feedback]
1. blurry arbitrary elements with full hd resolution, see on attached picture. No, it is not bad resolution of picture. It works like changing focus. When I hover it, it becomes clear for a moment and again blurry.  
2. no convenient movements, like in figma for instance when press space and use mouse to move around. I can move around only via touchpad
3. I did not got why login is _required_ for OSS
 
```
Version 2024.3.1
Build d0d9c52627d695574b546c24b096d87416ea8fc9
Electron 31.2.0
Chrome 126.0.6478.127
User Agent Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) enso/2024.3.1 Chrome/126.0.6478.127 Electron/31.2.0 Safari/537.36
```

![image](https://github.com/user-attachments/assets/962343ed-9166-41c0-8ee8-5c6470a20813)

**Comment by farmaazon** [comment]
Ad 1. It is already reported as https://github.com/enso-org/enso/issues/10717. Am I guessing right, that you have a fractional scaling?
@AdRiley probably the task is worth prioritizing.

Ad 2. We don't have space+mouse movement, but we have middle-mouse-button navigation (this one is also in Figma).

**Comment by RomanMIzulin** [comment]
yes, without fractional scaling it works clear
